src/gcodeExport.cpp

```diff
@@ -711,8 +733,17 @@ void GCodeExport::writeUnretractionAndPrime()
         }
         else
         {
-            current_e_value += extruder_attr[current_extruder].retraction_e_amount_current;
-            *output_stream << "G1 F" << PrecisionedDouble{1, extruder_attr[current_extruder].last_retraction_prime_speed * 60} << " " << extruder_attr[current_extruder].extruderCharacter << PrecisionedDouble{5, current_e_value} << new_line;
+            *output_stream << "G1 F" << PrecisionedDouble{1, extruder_attr[current_extruder].last_retraction_prime_speed * 60} << " " << extruder_attr[current_extruder].extruderCharacter;
+            if (relative_extrusion)
+            {
+                *output_stream << PrecisionedDouble{5, extruder_attr[current_extruder].retraction_e_amount_current + mm3ToE(prime_volume)} << new_line;
```

Comment:
This function now contains the expression `mm3ToE(prime_volume)` 3 times.
I'm not sure about compiler optimization and I might very well be nit picking here, but perhaps it would be better to just store the value somehwere in the beginning:
```
double prime_volume_e = mm3ToE(prime_volume);
```
and use that variable instead.